### PR TITLE
Debian patch for  building .deb file

### PR DESCRIPTION
**Pull request by onokatio**

### CODE REVIEW

File: contrib/debian/bitzeny-qt.desktop [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=BitZeny Core
+Comment=Connect to the BitZeny P2P Network
+Comment[de]=Verbinde mit dem BitXeny peer-to-peer Netzwerk
```
Comment (ghost) [inline contrib/debian/bitzeny-qt.desktop]: typo

File: contrib/debian/bitzeny-qt.desktop [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Comment[fr]=BitZeny, monnaie virtuelle cryptographique pair à pair
+Comment[tr]=BitZeny, eşten eşe kriptografik sanal para birimi
+Exec=bitzeny-qt %u
+Terminal=false
+Type=Application
+Icon=bitzeny128
```
Comment (ghost) [inline contrib/debian/bitzeny-qt.desktop:11]: missing icon

File: contrib/debian/bitzeny-qt.install [hunk 3]
```diff
@@ -0,0 +1,6 @@
+usr/local/bin/bitzeny-qt usr/bin
+share/pixmaps/bitzeny32.xpm usr/share/pixmaps
```
Comment (ghost) [inline contrib/debian/bitzeny-qt.install:2]: missing icon

File: contrib/debian/bitzeny-qt.install [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+share/pixmaps/bitzeny16.xpm usr/share/pixmaps
```
Comment (ghost) [inline contrib/debian/bitzeny-qt.install:3]: missing icon

File: contrib/debian/bitzeny-qt.install [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+share/pixmaps/bitzeny128.png usr/share/pixmaps
```
Comment (ghost) [inline contrib/debian/bitzeny-qt.install:4]: missing icon

**Review by ghost — CHANGES_REQUESTED** [review]
boo

**Comment by onokatio** [comment]
Fixed.

**Comment by ghost** [comment]
> This branch has conflicts that must be resolved

oh...

**Comment by onokatio** [comment]
Fix icon file name.
So conflict may be solved.

**Comment by ghost** [comment]
It has not been fixed yet...

**Comment by onokatio** [comment]
Rebased form z2.0.x branch.

**Comment by onokatio** [comment]
@hitobb  @shirako-jp plz review.

**Review by ghost — CHANGES_REQUESTED** [review]
This PR breaks compiling on Travis CI.
https://travis-ci.org/BitzenyCoreDevelopers/bitzeny/builds/340015664

**Comment by onokatio** [comment]
Fixed.

**Review by hitobb — DISMISSED** [review]
LGTM

**Review by ghost — CHANGES_REQUESTED** [review]
Please squash commits before merging into z2.0.x

**Comment by onokatio** [comment]
OK

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**